go/bundle/bundle.go

```diff
@@ -0,0 +1,237 @@
+package bundle
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/WICG/webpackage/go/signedexchange"
+	"github.com/WICG/webpackage/go/signedexchange/cbor"
+)
+
+var HeaderMagicBytes = []byte{0x84, 0x48, 0xf0, 0x9f, 0x8c, 0x90, 0xf0, 0x9f, 0x93, 0xa6}
+
+const FooterLength = 9
+
+type Input struct {
+	Exchanges []*signedexchange.Exchange
```

Comment:
Do you have a plan to add members to the input? If no, just using slice would be enough.

Reply:
Yes. The implementation only encodes "index","responses" sections atm, but there are other sections (such as manifest)